src/Microsoft.AspNetCore.Razor/Parser/TagHelpers/TagHelperAttributeNode.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Parser.SyntaxTree;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Razor.Parser.TagHelpers
+{
+    public class TagHelperAttributeNode
+    {
+        public TagHelperAttributeNode(string name, SyntaxTreeNode value)
+            : this(name, value, HtmlAttributeStructure.DoubleQuotedValue)
```

Comment:
Yes to a `TagHelperAttributeNode(string name)` overload; no to this one. Whenever there's an attribute value, structure choice should be explicit.